datarobot_batch_scoring/batch_scoring.py

```diff
@@ -677,6 +679,23 @@ def batch_generator_args(self):
         return args
 
 
+def is_redirect(req, ui):
```

Comment:
I think `warn_if_redirect` would be a better name for this. `is_redirect` sounds like it returns a boolean (whereas your function is called only for the side effects)